meilisearch/index.py

```diff
@@ -242,21 +241,16 @@ def search(self, query, opt_params=None):
         # Query parameters parsing
         if opt_params is None:
             opt_params = {}
-        for key in opt_params:
-            if key in ('facetsDistribution', 'facetFilters'):
-                opt_params[key] = json.dumps(opt_params[key])
-            elif isinstance(opt_params[key], list):
-                opt_params[key] = ','.join(opt_params[key])
         params = {
```

Comment:
```suggestion
        body = {
```

```suggestion
       data = {
```

Just better naming because those are not params anymore. But It's up to you if you want to change it.